cli/src/jobs_runner/jobs_state.rs

```diff
@@ -11,6 +11,10 @@ use crate::tracker::get_tracker;
 /// Duration to wait for jobs when cancellation is invoked.
 pub const TIMEOUT_DURATION: Duration = Duration::from_secs(2);
 
+/// Duration to wait after starting the shutdown process, to force the program to exit if it's
+/// still active after the given has passed.
+pub const FORCE_EXIT_DURATION: Duration = Duration::from_secs(2);
```

Comment:
I would give it some more time, maybe 5 seconds. On virus scanner days any machine can have some latency of 1-2 sec sporadically..

Reply:
The whole time here is actually 4 seconds because the app will wait for the first TIMEOUT_DURATION (2 Seconds) then this 2, but I think it would make sense to increase it more because I was trying it on my beefy machine :+1: